.github/workflows/test.yaml

```diff
@@ -27,15 +27,22 @@ jobs:
         uses: ./.github/actions/dev-env-setup
       - run: yarn install --immutable
         working-directory: ./bciers
-  yarn-test:
+  nx-tests:
     needs: install-dev-tools
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./bciers
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
       - name: dev env setup
         uses: ./.github/actions/dev-env-setup
-      - run: yarn test
-        working-directory: ./client
+      - name: Run Nx Tests
+        run: yarn nx run-many -t test --skipNxCache
```

Comment:
Let's get a card to look at the caching, I think this is one of the places that Nx can really improve our CI run times